tests/test_integration.py

```diff
@@ -278,3 +278,52 @@ def test_pex_path_arg():
     stdout, rc = run_simple_pex(pex_out_path, [test_file_path])
     assert rc == 0
     assert stdout == b'Success!\n'
+
+
+def test_pex_path_in_pex_info_and_env():
+  with temporary_dir() as output_dir:
+
+    # create 2 pex files for PEX-INFO pex_path
+    pex1_path = os.path.join(output_dir, 'pex1.pex')
+    res1 = run_pex_command(['--disable-cache', 'requests', '-o', pex1_path])
+    res1.assert_success()
+    pex2_path = os.path.join(output_dir, 'pex2.pex')
+    res2 = run_pex_command(['--disable-cache', 'flask', '-o', pex2_path])
+    res2.assert_success()
+    pex_path = ':'.join(os.path.join(output_dir, name) for name in ('pex1.pex', 'pex2.pex'))
+
+    # create a pex for environment PEX_PATH
+    pex3_path = os.path.join(output_dir, 'pex3.pex')
+    res3 = run_pex_command(['--disable-cache', 'sqlalchemy', '-o', pex3_path])
```

Comment:
also, iirc `sqlalchemy` is a pretty heavy dep. how about something lighter, like `bitarray`, `enum34`, `mox` etc?